releasenotes/notes/fix_11143-d32a262538873a9d.yaml

```
@@ -0,0 +1,5 @@
+deprecations:
+  - |
+    The functions :func:`.num_to_latex_ket` and :func:`.numbers_to_latex_terms` 
+    have been removed. For similar functionality, see sympy's 
+    ``nsimplify`` and ``latex`` functions.
```

I would suggest something along these lines instead:

```suggestion
upgrade:
  - |
    The functions ``qiskit.visualization.state_visualization.num_to_latex_ket`` and ``qiskit.visualization.state_visualization.numbers_to_latex_terms`` 
    have been removed, as they were removed in Qiskit 0.40 (released in Jan 2023). For similar functionality, see sympy's ``nsimplify`` and ``latex`` functions.
```

---

@1ucian0 can you please suggest how to preview/tests release notes before commiting as `sphinx` docs quite confusing. For instance, is there any difference between `qiskit.visualization`(single backticks) ``qiskit.visualization`` (double backticks).

---

@Raghav-Bell we have an open PR that lists all the options to build docs locally: https://github.com/Qiskit/qiskit/pull/11073. You can also download the CI artifacts from the docs build run clicking on `details` ->  `View more details on Azure Pipelines` -> find artifacts and download. As for your second question:

- single backticks  -> inline code in markdown language (used by GitHub, for example)
- double backticks  ->  inline code in rst (used by Sphinx)
- something like  ``:func:`.num_to_latex_ket``` (ignore double backticks) -> link to corresponding API doc entry in Sphinx

When you are documenting a removal, the corresponding API doc entry should not exist anymore, because the code is no longer there, so instead of the usual links we just use the double backticks to make it look like inline code.